focused/security/restBasicAuthDBStore/src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app version="5.0"
+	xmlns="https://jakarta.ee/xml/ns/jakartaee"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-app_5_0.xsd">
```

Comment:
Should be 6.0 for EE10

Reply:
By the way, formatting and indentation is inconsistent across all the xml files. Perhaps we want to set up a style guide and formatting rules for more consistency and professionality.